scripts/v1alpha2/create_controlplane.sh

```diff
@@ -5,4 +5,4 @@ METAL3_DIR="$(dirname "$(readlink -f "${0}")")/../.."
 # shellcheck disable=SC1090
 source "${METAL3_DIR}/lib/common.sh"
 
-make_v1alpha2_machine controlplane | kubectl apply -n metal3 -f -
+make_v1alphaX_machine controlplane 2 | kubectl apply -n metal3 -f -
```

Comment:
you could give the CR path here instead of "2" that is not explicit

Reply:
done